<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-1-g15f4949  Build Version: 1.6.7594.29634
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="robot_sim">
  <link
    name="base_footprint">
    <inertial>
      <origin
        xyz="0.00671991635536837 -3.15645549871153E-05 0.1375"
        rpy="0 0 0" />
      <mass
        value="0.148429203673205" />
      <inertia
        ixx="0.000177786135907738"
        ixy="1.3648480107125E-20"
        ixz="-5.91994304686235E-22"
        iyy="0.000764231380527697"
        iyz="5.98618839213826E-22"
        izz="0.00094139906142013" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <!-- <geometry>
        <mesh
          filename="package://robot_sim/meshes/base_footprint.STL" />
      </geometry> -->
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <!-- <geometry>
        <mesh
          filename="package://robot_sim/meshes/base_footprint.STL" />
      </geometry> -->
    </collision>
  </link>
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="-0.00488560336421057 -0.00113618674554441 0.00936292689878374"
        rpy="0 0 0" />
      <mass
        value="0.597842637267807" />
      <inertia
        ixx="0.000447764687219218"
        ixy="1.87857105821224E-19"
        ixz="2.78328487565125E-21"
        iyy="0.001584315084448"
        iyz="-2.960158484516E-20"
        izz="0.00198084843982028" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot_sim/meshes/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot_sim/meshes/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="base_joint"
    type="fixed">
    <origin
      xyz="0.0067199 0.0011046 0.085"
      rpy="0 0 0" />
    <parent
      link="base_footprint" />
    <child
      link="base_link" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="wheel_front_left_link">
    <inertial>
      <origin
        xyz="-1.3675E-06 -6.2077E-06 -4.5411E-06"
        rpy="0 0 0" />
      <mass
        value="0.19171" />
      <inertia
        ixx="0.00013676"
        ixy="-9.5175E-09"
        ixz="-1.5304E-09"
        iyy="0.00022818"
        iyz="-6.6419E-09"
        izz="0.00013676" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot_sim/meshes/wheel_front_left_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot_sim/meshes/wheel_front_left_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="wheel_front_left_joint"
    type="continuous">
    <origin
      xyz="0.075 0.096581 -0.037"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="wheel_front_left_link" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="wheel_front_right_link">
    <inertial>
      <origin
        xyz="-1.3952E-06 7.9271E-06 -4.5555E-06"
        rpy="0 0 0" />
      <mass
        value="0.19167" />
      <inertia
        ixx="0.00013674"
        ixy="8.5942E-09"
        ixz="1.842E-09"
        iyy="0.00022815"
        iyz="7.22E-09"
        izz="0.00013674" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot_sim/meshes/wheel_front_right_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot_sim/meshes/wheel_front_right_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="wheel_front_right_joint"
    type="continuous">
    <origin
      xyz="0.075 -0.098854 -0.037"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="wheel_front_right_link" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="wheel_back_left_link">
    <inertial>
      <origin
        xyz="1.3952E-06 -7.9271E-06 -4.5555E-06"
        rpy="0 0 0" />
      <mass
        value="0.19167" />
      <inertia
        ixx="0.00013674"
        ixy="8.5942E-09"
        ixz="-1.842E-09"
        iyy="0.00022815"
        iyz="-7.22E-09"
        izz="0.00013674" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot_sim/meshes/wheel_back_left_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot_sim/meshes/wheel_back_left_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="wheel_back_left_joint"
    type="continuous">
    <origin
      xyz="-0.09 0.096581 -0.037"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="wheel_back_left_link" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="wheel_back_right_link">
    <inertial>
      <origin
        xyz="1.3675E-06 6.2077E-06 -4.5411E-06"
        rpy="0 0 0" />
      <mass
        value="0.19171" />
      <inertia
        ixx="0.00013676"
        ixy="-9.5175E-09"
        ixz="1.5304E-09"
        iyy="0.00022818"
        iyz="6.6418E-09"
        izz="0.00013676" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot_sim/meshes/wheel_back_right_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.79216 0.81961 0.93333 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot_sim/meshes/wheel_back_right_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="wheel_back_right_joint"
    type="continuous">
    <origin
      xyz="-0.09 -0.098854 -0.037"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="wheel_back_right_link" />
    <axis
      xyz="0 1 0" />
  </joint>
  <link
    name="camera_link">
    <inertial>
      <origin
        xyz="0.00118426840072709 0.000354802197291548 -0.0033111064543756"
        rpy="0 0 0" />
      <mass
        value="0.0813917545444659" />
      <inertia
        ixx="0.000137853739097722"
        ixy="2.15870431819302E-07"
        ixz="-1.36206428007122E-06"
        iyy="1.15294219486457E-05"
        iyz="-8.20794085614286E-07"
        izz="0.000133078432654531" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot_sim/meshes/camera_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://robot_sim/meshes/camera_link.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="camera_joint"
    type="fixed">
    <origin
      xyz="0.08163 -0.0012919 0.08"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="camera_link" />
    <axis
      xyz="0 0 0" />
  </joint>
  <link
    name="lidar_link">
    <inertial>
      <origin
        xyz="0.00991135942805239 0.00216264107596675 -0.00892040118258941"
        rpy="0 0 0" />
      <mass
        value="0.0196171588596213" />
      <inertia
        ixx="7.69141187979746E-06"
        ixy="-1.9553426215541E-08"
        ixz="8.71698572703788E-08"
        iyy="6.49320734988539E-06"
        iyz="-2.66260948561593E-08"
        izz="1.34575412495693E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <!-- <geometry>
        <mesh
          filename="package://robot_sim/meshes/lidar_link.STL" />
      </geometry> -->
      <material
        name="">
        <color
          rgba="1 1 1 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <!-- <geometry>
        <mesh
          filename="package://robot_sim/meshes/lidar_link.STL" />
      </geometry> -->
    </collision>
  </link>
  <joint
    name="lidar_joint"
    type="fixed">
    <origin
      xyz="-0.018284 -0.003868 0.11"
      rpy="0 0 0" />
    <parent
      link="base_link" />
    <child
      link="lidar_link" />
    <axis
      xyz="0 0 0" />
  </joint>
</robot>